clustering/jgroups/WFLY-20444_Upgrade_JGroups_AWS_to_3_0_1_Final.adoc

```diff
@@ -0,0 +1,119 @@
+---
+categories:
+  - clustering
+  - cloud
+stability-level: default
+issue: https://github.com/wildfly/wildfly-proposals/issues/690
+feature-team:
+ developer: rhusar
+ sme:
+  - rhusar
```

Comment:
@rhusar No double-dipping. :) Perhaps you can go beat on Paul get him to sign onto this, since he already approved the main PR.

Reply:
Resolving, I asked @pferraro to kindly volunteer last week :)